azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/error/ErrorCode.kt

```diff
@@ -14,6 +14,7 @@ internal class ErrorCode : ExpandableStringEnum<ErrorCode?>() {
         val TURN_CAMERA_OFF_FAILED = fromString("turnCameraOffFailed")
         val TURN_MIC_ON_FAILED = fromString("turnMicOnFailed")
         val TURN_MIC_OFF_FAILED = fromString("turnMicOffFailed")
+        val UNKNOWN_ERROR = fromString("unknownError")
```

Comment:
do we have discussion with iOs team to sync on this error?

Reply:
Yes. iOS team and we are on sync with introducing the Unknown Error. 
Basically, we will use this error whenever the composite runs to an ambiguous state which happens due to any system related issue of the device such as this type of error (getting error on device manager)